pkg/kubeapiserver/admission/config.go

```diff
@@ -44,8 +45,8 @@ type Config struct {
 }
 
 // New sets up the plugins and admission start hooks needed for admission
-func (c *Config) New(proxyTransport *http.Transport, serviceResolver webhook.ServiceResolver) ([]admission.PluginInitializer, genericapiserver.PostStartHookFunc, error) {
-	webhookAuthResolverWrapper := webhook.NewDefaultAuthenticationInfoResolverWrapper(proxyTransport, c.LoopbackClientConfig)
+func (c *Config) New(proxyTransport *http.Transport, egressLookup egressselector.Lookup, serviceResolver webhook.ServiceResolver) ([]admission.PluginInitializer, genericapiserver.PostStartHookFunc, error) {
```

Comment:
I'd prefer seeing the entire `*EgressSelector` wired here.

Reply:
The up and down side of passing through the EgressSelector is it would allow us to call methods beyond just Lookup, if and when we get such methods. Given that we control this code and its usage, its not clear to me that we need to protect ourselves from things like paths modifying the map (please don't). Given that I would say this is a decent future proofing suggestion.